usermods/Internal_Temperature_v2/usermod_internal_temperature.h

```diff
@@ -0,0 +1,134 @@
+#pragma once
+
+#include "wled.h"
+
+class InternalTemperatureUsermod : public Usermod
+{
+
+private:
+  unsigned long loopInterval;
```

Comment:
loopInterval should have a default value.
I would suggest something like 15 seconds.

Reply:
It has a default value, see line 99.